tb_device_mqtt.py

```diff
@@ -241,16 +315,25 @@ def _on_connect(self, client, userdata, flags, result_code, *extra_params):
             self._client.subscribe(ATTRIBUTES_TOPIC + "/response/+", qos=self.quality_of_service)
             self._client.subscribe(RPC_REQUEST_TOPIC + '+', qos=self.quality_of_service)
             self._client.subscribe(RPC_RESPONSE_TOPIC + '+', qos=self.quality_of_service)
+            self._client.subscribe("v2/fw/response/+")
+            self.send_telemetry(self.current_firmware_info)
+            self.request_firmware_info()
```

Comment:
Is it necessary to request firmware info on every connect, what if we don't want update the client, but want to have ability to do this?